src/core/public/legacy/legacy_service.ts

```diff
@@ -172,20 +173,6 @@ export class LegacyPlatformService {
     // clear the inner html of the root angular element
     this.targetDomElement.textContent = '';
   }
-
-  private loadBootstrapModule(): BootstrapModule {
-    if (this.params.useLegacyTestHarness) {
-      // wrapped in NODE_ENV check so the `ui/test_harness` module
-      // is not included in the distributable
-      if (process.env.IS_KIBANA_DISTRIBUTABLE !== 'true') {
-        return require('ui/test_harness');
-      }
-
-      throw new Error('tests bundle is not available in the distributable');
-    }
-
-    return require('ui/chrome');
-  }
```

Comment:
I see this logic moved to a split of `requireLegacyBootstrapModule` value between `src/legacy/core_plugins/tests_bundle/tests_entry_template.js ` and `src/legacy/ui/ui_bundles/app_entry_template.js` 

The previous condition was `this.params.useLegacyTestHarness` + `process.env.IS_KIBANA_DISTRIBUTABLE !== 'true'`. Is that condition the exact same one for whereas we are using the test entry template or the app one?

Reply:
I still need to see if checking `process.env.IS_KIBANA_DISTRIBUTABLE !== 'true'` is needed anymore or not. AFAICT this was only necessary when building the `test_entry_template` which _shouldn't_ be done in production, but I need to verify.